src/Compilers/CSharp/Test/Syntax/Parsing/DeconstructionTests.cs

```diff
@@ -2071,7 +2071,6 @@ struct ValueTuple<T1, T2>
         public ValueTuple(T1 item1, T2 item2) { this.Item1 = item1; this.Item2 = item2; }
     }
 }";
-            // the duplicate reporting of CS8136 below is due to open issue https://github.com/dotnet/roslyn/issues/12905
```

Comment:
Perhaps. It needs to be tested in other contexts as well. I suspect there is an issue in `foreach`.

---
In reply to: [90307970](https://github.com/dotnet/roslyn/pull/15548#discussion_r90307970) [](ancestors = 90307970)